routes/get_ticket_artifact.go

```diff
@@ -0,0 +1,36 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/csci4950tgt/api/models"
+	"github.com/csci4950tgt/api/util"
+	"github.com/gorilla/mux"
+)
+
+func GetTicketArtifact(w http.ResponseWriter, r *http.Request) {
+	// Get variables from route handler
+	vars := mux.Vars(r) // get dynamic variables from mux handler
+	artifact := vars["artifact"]
+	id, err := strconv.Atoi(vars["id"]) // get integer "id" from vars
+
+	if err != nil {
+		util.WriteHttpErrorCode(w, http.StatusBadRequest, "Missing required parameter: id.")
+
+		return
+	}
+
+	// TODO: Actually get desired artifact from ticket
+
+	// Initialize Response
+	msg := fmt.Sprintf("Not yet implemented... Id: %d, Artifact: %s", id, artifact)
+	res := models.Response{
```

Comment:
When this gets implemented, the response will be a binary (`byte[]`) instead of utf8 JSON, because we can't embed the file object inside a JSON field (without base64, which adds a lot of overhead). As long as the frontend doesn't rely on a specific response type for this endpoint, that's fine.

Reply:
True, I think we can add a file_artifacts field to the models.Response struct when implementing that. We'll also need to add a tickets array field in the future for getting information about multiple tickets